events/loginEvent.py

```diff
@@ -130,15 +139,6 @@ def handle(
                 glob.verifiedCache[str(userID)] = 0
                 raise exceptions.loginBannedException()
 
-        # Save HWID in db for multiaccount detection
-        hwAllowed = userUtils.logHardware(userID, clientData, firstLogin)
```

Comment:
LOL wtf did it not log before if the user got banned?

Reply:
indeed it did not :pensive: